src/components/attributes/CommonComponents.js

```diff
@@ -15,41 +17,61 @@ function changeId (entity, componentName, propertyName, value) {
   }
 }
 
-const CommonComponents = props => {
-  const entity = props.entity;
-  const components = entity ? props.entity.components : {};
-  if (!entity) { return <div></div>; }
-  return (
-    <Collapsible>
-      <div className='collapsible-header'>
-        <span>Common</span>
-      </div>
-      <div className='collapsible-content'>
-        <div className='row'>
-          <span className='value tagName'><code>&lt;{entity.tagName.toLowerCase()}&gt;</code></span>
+class CommonComponents extends React.Component {
+
+  componentDidMount () {
+    var self = this;
+    var clipboard = new Clipboard('[data-action="copy-entity-to-clipboard"]', {
+      text: function (trigger) {
+        return self.props.entity.outerHTML;
+      }
+    });
+    clipboard.on('error', function (e) {
+      // @todo Show the error on the UI
+    });
+  }
+
+  render () {
+    const entity = this.props.entity;
+    const components = entity ? this.props.entity.components : {};
+    if (!entity) { return <div></div>; }
+    return (
+      <Collapsible>
+        <div className='collapsible-header'>
+          <span>Common</span>
+          <div>
+            <a href='#' title='Copy entity HTML to clipboard' data-action='copy-entity-to-clipboard'
+              className='button fa fa-clipboard' onClick={event => event.stopPropagation()}></a>
+          </div>
         </div>
-        <div className='row'>
-          <span className='text'>ID</span>
-          <InputWidget onChange={changeId} entity={entity} name='id' value={entity.id}/>
+        <div className='collapsible-content'>
+          <div className='row'>
+            <span className='value tagName'><code>&lt;{entity.tagName.toLowerCase()}&gt;</code></span>
+          </div>
+          <div className='row'>
+            <span className='text'>ID</span>
+            <InputWidget onChange={changeId} entity={entity} name='id' value={entity.id}/>
+          </div>
+          {
+            Object.keys(components).filter(function (key) {
+              return ['visible', 'position', 'scale', 'rotation'].indexOf(key) !== -1;
```

Comment:
These components are also defined in ```Attributes.js```, perhaps move them to a constants or share them.